samples/java-eventsourced-counter/src/test/java/com/example/domain/CounterJournalToTopicActionTest.java

```diff
@@ -0,0 +1,25 @@
+package com.example.domain;
+
+import com.akkaserverless.javasdk.action.Action;
+import com.akkaserverless.javasdk.impl.action.ActionEffectImpl;
+import com.akkaserverless.javasdk.testkit.ActionResult;
+import com.example.actions.CounterTopicApi;
+import com.example.actions.CounterJournalToTopicAction;
+import com.google.protobuf.Empty;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class CounterJournalToTopicActionTest {
+
+  @Test
+  public void increaseTest() {
+    CounterJournalToTopicActionTestKit testKit = CounterJournalToTopicActionTestKit.of(CounterJournalToTopicAction::new);
+    int valueToincrease = 1;
+    ActionResult<CounterTopicApi.Increased> result = testKit.increase(CounterDomain.ValueIncreased.newBuilder().setValue(valueToincrease).build());
+    assertTrue(result.isReply());
+    CounterTopicApi.Increased replyMessage = result.getReplyMessage();
+    assertEquals(replyMessage.getValue(),valueToincrease);
```

Comment:
for `assertEquals` the expect value is supposed to be the first parameter

Reply:
done